package-lock.json

```diff
@@ -10288,10 +10288,25 @@
       "resolved": "https://registry.npmjs.org/eth-sig-util/-/eth-sig-util-2.0.2.tgz",
       "integrity": "sha512-tB6E8jf/aZQ943bo3+iojl8xRe3Jzcl+9OT6v8K7kWis6PdIX19SB2vYvN849cB9G9m/XLjYFK381SgdbsnpTA==",
       "requires": {
+<<<<<<< HEAD
```

Comment:
We'll want to remove these merge markers

Reply:
whoooops, missed that one :(

Reply:
Fixed. Running `npm i` fixes package-lock conflicts.